MSBuildGitHash/build/MSBuildGitHash.targets

```diff
@@ -2,7 +2,9 @@
 
   <PropertyGroup>
     <IncludeMSBuildGitHashMetadata Condition="'$(IncludeMSBuildGitHashMetadata)' == ''">true</IncludeMSBuildGitHashMetadata>
+    <IncludeMSBuildGitDescribeMetadata Condition="'$(IncludeMSBuildGitDescribeMetadata)' == ''">true</IncludeMSBuildGitDescribeMetadata>
     <IncludeMSBuildGitHashInfoVersion Condition="'$(IncludeMSBuildGitHashInfoVersion)' == ''">true</IncludeMSBuildGitHashInfoVersion>
+    <IncludeMSBuildGitDescribeAssemblyVersion Condition="'$(IncludeMSBuildGitDescribeAssemblyVersion)' == ''">false</IncludeMSBuildGitDescribeAssemblyVersion>
```

Comment:
this is kinda weird cause it's unclear what would happen if this and IncludeMSBuildGitHashInfoVersion  were both true, i wasn't expecting to go this far

Reply:
So add it to the custom attributes but NOT the assembly meta data?

Reply:
yeah cause u can only have 1 be the assembly version...i think a further enhancement would be to deprecate IncludeMSBuildGitHashInfoVersion  and instead have it be more of a map as to what value should be used maybe

Reply:
but dont' want to do anything related to that now